spec/unit/puppet/provider/network_vlan/cisco_ios_spec.rb

```diff
@@ -8,7 +8,19 @@ def self.load_test_data
     PuppetX::CiscoIOS::Utility.load_yaml(File.expand_path(__dir__) + '/test_data.yaml', false)
   end
 
+  def self.action_name(action)
```

Comment:
This method does not appear to be called

Reply:
deleted